apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -240,14 +240,11 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
                   ))
             }
             {
-              // Alternative travel ban restrictions
-              (workingCase.decision ===
-                CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN ||
-                (workingCase.type === CaseType.TRAVEL_BAN &&
-                  (workingCase.decision === CaseDecision.ACCEPTING ||
-                    workingCase.decision ===
-                      CaseDecision.ACCEPTING_PARTIALLY))) &&
-                workingCase.custodyRestrictions
+              // Travel ban restrictions
```

Comment:
Isn't there some inconsistency for custody cases that get an alternative travel ban ruling?

The court record is prefilled with CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_REQUIRE_NOTIFICATION and CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_CONFISCATE_PASSPORT, but we show nothing here.

Reply:
I asked about this on Slack and Baldur said we should not show anything here

<img width="1215" alt="Screenshot 2022-01-07 at 10 32 25" src="https://user-images.githubusercontent.com/3789875/148531258-1eaa9dfa-e065-4c75-8482-8ae5d06539f7.png">